test/e2e/create_enable_force_delete_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e_test
+
+import (
+	"context"
+	"time"
+
+	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+var _ = Describe("Registry Cache Extension Tests", func() {
+	parentCtx := context.Background()
+
+	f := defaultShootCreationFramework()
+	f.Shoot = defaultShoot("e2e-default")
+
+	It("should create Shoot, enable the registry-cache extension, force delete Shoot", func() {
+		By("Create Shoot")
+		ctx, cancel := context.WithTimeout(parentCtx, 15*time.Minute)
+		defer cancel()
+		Expect(f.CreateShootAndWaitForCreation(ctx, false)).To(Succeed())
+		f.Verify()
+
+		By("Enable the registry-cache extension")
+		ctx, cancel = context.WithTimeout(parentCtx, 10*time.Minute)
+		defer cancel()
+		Expect(f.UpdateShoot(ctx, f.Shoot, func(shoot *gardencorev1beta1.Shoot) error {
+			size := resource.MustParse("2Gi")
+			extension := registryCacheExtension("docker.io", &size)
+			shoot.Spec.Extensions = []gardencorev1beta1.Extension{extension}
+
+			return nil
+		})).To(Succeed())
+
+		By("Verify registry-cache works")
+		verifyRegistryCache(parentCtx, f.Logger, f.ShootFramework.ShootClient, "docker.io", nginxImageWithDigest)
+
+		By("Force Delete Shoot")
+		ctx, cancel = context.WithTimeout(parentCtx, 10*time.Minute)
```

Comment:
```suggestion
		ctx, cancel = context.WithTimeout(parentCtx, 15*time.Minute)
```

Reply:
Force delete should takes around 1-2 minutes only. So 10 minutes should be fine.

Reply:
> Force delete should takes around 1-2 minutes only. So 10 minutes should be fine. WDYT?

The 15 min context timeout for shoot deletion is used in other e2e tests, but if force delete is faster then it is OK